apps/researcher/src/messages/nl/messages.json

```diff
@@ -4,27 +4,15 @@
   },
   "Home": {
     "title": "{totalDatasets, plural, =0 {0 Objecten} =1 {1 Object} other {# Objecten}}",
-    "search": "Zoeken",
-    "ownersFilter": "Eigenaren",
-    "typesFilter": "Types",
-    "subjectsFilter": "Onderwerpen",
     "noResults": "Er zijn geen resultaten",
-    "sortRelevanceDesc": "Relevantie",
-    "sortNameAsc": "Naam - Oplopend",
-    "sortNameDesc": "Naam - Aflopend",
     "fetchError": "Er is een probleem opgetreden bij het ophalen van de objecten. Laad de pagina opnieuw om het opnieuw te proberen.",
-    "filters": "Filters",
-    "clearAllFilters": "Alles verwijderen",
-    "expandFilter": "Uitklappen",
-    "filterSearchPlaceholder": "Filteren op {filterName}",
-    "accessibilityTypeToFilter": "Typ om op tekst te filteren",
-    "accessibilitySelectToChangeOrder": "Selecteer om de volgorde van het resultaat te wijzigen",
-    "accessibilitySelectToFilter": "Selecteer een selectievakje om te filteren"
+    "filters": "Filters"
   },
-  "Paginator": {
-    "results": "Resultaat <number>{start}</number> tot <number>{end}</number> van de <number>{totalCount}</number>",
-    "previous": "Vorige",
-    "next": "Volgende"
+  "Persons": {
+    "title": "{totalPersons, plural, =0 {0 personen} =1 {1 persoon} other {# personen}}",
+    "noResults": "Er zijn geen resultaten",
+    "fetchError": "Er is een probleem opgetreden bij het ophalen van de personen. Laad de pagina opnieuw om het opnieuw te proberen.",
```

Comment:
Instead of 'personen': 'resultaten' (to be in sync with `noResults` - 'Er zijn geen resultaten')?